context.go

```diff
@@ -36,10 +38,14 @@ func LoadContext(confName string, confPaths []string) (*Context, error) {
 	var err error
 	var conf *Config
 	for _, confPath := range confPaths {
-		conf, err = ReadDefault(path.Join(confPath, confName))
+		filePath := path.Join(confPath, confName)
+		conf, err = ReadDefault(filePath)
 		if err == nil {
```

Comment:
Can you please change it to below:
```go
if err != nil {
	return nil, fmt.Errorf("%v: %v", filePath, err)
} 

return &Context{config: conf}, nil
```

While reviewing this PR, I noticed `LoadContext` method only loads a first `confPath` from `confPaths`. So It needs an attention. I will create an issue for that separately.